WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationCommentDetailViewController.swift

```diff
@@ -149,7 +149,7 @@ private extension NotificationCommentDetailViewController {
                                                                       notificationDelegate: notificationDelegate,
                                                                       managedObjectContext: managedObjectContext)
 
-            commentDetailViewController.view.frame = view.frame
+            commentDetailViewController.view.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
I should have this tattooed on the back of my hand. I can't believe I forgot this. 🤦